src/fdk_rdf_parser/classes/model_element.py

```diff
@@ -22,7 +22,9 @@ class ModelCodeElement:
     altLabel: Optional[Dict[str, str]] = None
 
     def __lt__(self: Any, other: Any) -> bool:
-        return self.identifier < other.identifier
+        self_identifier = self.identifier if self.identifier else ""
+        other_identifier = other.identifier if other.identifier else ""
+        return self_identifier < other_identifier
```

Comment:
'identifier' er defintert som string og da sammenligner du ascii-verdier. Tror du da kan risikere at '1999' < '5'. Du unngår det hvis du sier int(self_identifier) < int(other_identifier). Da må du vel også ha else "0" foran hvis ikke eksisterer.

Reply:
identifier vil ikke nødvendigvis være tall, er vel så sannsynlig at det er snakk om GUID eller lignende.

Reply:
Hvorfor sammenligne to strenger?

Reply:
for å kunne bruke sort(), identifier var det eneste feltet som var obligatorisk